Source/Scene/GroundPrimitiveBatch.js

```diff
@@ -0,0 +1,997 @@
+
+        for (i = 0; i < countsLength; ++i) {
+            color = batchTable.getColor(batchIds[i], scratchColor);
+            rgba = color.toRgba();
+
+            buffer = colorToBuffers[rgba];
+            var positionOffset = buffer.offset;
+            var positionIndex = positionOffset * 3;
+            var colorIndex = positionOffset * 4;
+            var batchIdIndex = positionOffset;
+
+            var polygonOffset = offsets[i];
+            var polygonCount = counts[i];
+            var batchId = batchIds[i];
+
+            var minLat = Number.POSITIVE_INFINITY;
+            var maxLat = Number.NEGATIVE_INFINITY;
+            var minLon = Number.POSITIVE_INFINITY;
+            var maxLon = Number.NEGATIVE_INFINITY;
+
+            for (j = 0; j < polygonCount; ++j) {
+                var encodedPosition = Cartesian3.unpack(positions, polygonOffset * 3 + j * 3, scratchEncodedPosition);
+                var rtcPosition = Matrix4.multiplyByPoint(decodeMatrix, encodedPosition, encodedPosition);
+                var position = Cartesian3.add(rtcPosition, center, rtcPosition);
+
+                var carto = ellipsoid.cartesianToCartographic(position, scratchBVCartographic);
+                var lat = carto.latitude;
+                var lon = carto.longitude;
+
+                minLat = Math.min(lat, minLat);
+                maxLat = Math.max(lat, maxLat);
+                minLon = Math.min(lon, minLon);
+                maxLon = Math.max(lon, maxLon);
+
+                var normal = ellipsoid.geodeticSurfaceNormal(position, scratchNormal);
+                var scaledPosition = ellipsoid.scaleToGeodeticSurface(position, position);
+                var scaledNormal = Cartesian3.multiplyByScalar(normal, minHeight, scratchScaledNormal);
+                var minHeightPosition = Cartesian3.add(scaledPosition, scaledNormal, scratchMinHeightPosition);
+
+                scaledNormal = Cartesian3.multiplyByScalar(normal, maxHeight, scaledNormal);
+                var maxHeightPosition = Cartesian3.add(scaledPosition, scaledNormal, scratchMaxHeightPosition);
+
+                Cartesian3.subtract(maxHeightPosition, center, maxHeightPosition);
+                Cartesian3.subtract(minHeightPosition, center, minHeightPosition);
+
+                Cartesian3.pack(maxHeightPosition, batchedPositions, positionIndex);
+                Cartesian3.pack(minHeightPosition, batchedPositions, positionIndex + 3);
+
+                batchedIds[batchIdIndex] = batchId;
+                batchedIds[batchIdIndex + 1] = batchId;
+
+                positionIndex += 6;
+                colorIndex += 8;
+                batchIdIndex += 2;
+            }
+
+            var rectangle = scratchBVRectangle;
+            rectangle.west = minLon;
+            rectangle.east = maxLon;
+            rectangle.south = minLat;
+            rectangle.north = maxLat;
+
+            boundingVolumes[i] = OrientedBoundingBox.fromRectangle(rectangle, minHeight, maxHeight, ellipsoid);
```

Comment:
Is this only for picking?  If so, it might be worth just putting a `PERFORMANCE_IDEA` comment and then addressing this later since we'll have to think through increasing the payload size vs. computing the tight fit at runtime.

CC https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/127